### PR TITLE
discount: add with-fenced-code options.

### PR DESCRIPTION
**Pull request by Sixeight**
This Pull Request adds an option to Pandoc-style fenced code blocks support to discount.

see `Fenced code blocks` at http://www.pell.portland.or.us/~orc/Code/markdown/ for more info.

thanks :)

### CODE REVIEW

File: Library/Formula/discount.rb [hunk 1]
```diff
@@ -5,14 +5,20 @@ class Discount < Formula
...
   sha1 '517bcf7409d8c02b3e57f51264b2e110f8a03120'
 
+  option "with-fenced-code", "Enable Pandoc-style fenced code blocks."
+
   conflicts_with 'markdown',
     :because => 'both discount and markdown ship a `markdown` executable.'
 
   def install
+    opts = []
+    opts << "--with-fenced-code" if build.include? "with-fenced-code"
```
Comment (mikemcquaid) [inline Library/Formula/discount.rb]: Use `build.with? "fenced-code"`. Also, any reason to not enable this by default?

File: Library/Formula/discount.rb [hunk 2]
```diff
@@ -5,14 +5,20 @@ class Discount < Formula
   url 'http://www.pell.portland.or.us/~orc/Code/discount/discount-2.1.7.tar.bz2'
   sha1 '517bcf7409d8c02b3e57f51264b2e110f8a03120'
 
+  option "with-fenced-code", "Enable Pandoc-style fenced code blocks."
+
   conflicts_with 'markdown',
     :because => 'both discount and markdown ship a `markdown` executable.'
 
   def install
+    opts = []
```
Comment (mikemcquaid) [inline Library/Formula/discount.rb]: Name this `args` and add the other `./configure` arguments to it first, thanks!

Comment (Sixeight) [inline Library/Formula/discount.rb]: ok I'll fix it!

File: Library/Formula/discount.rb (hunk 1 above)
Comment (Sixeight) [inline Library/Formula/discount.rb]: I'm sorry for wrong use.
Because this formula hasn't this configure option, I don't want to change current behavior.

**Comment by Sixeight** [comment]
@mikemcquaid Thank you for your response. I've corrected according to your advice.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**